src/Casino/state/thunks.ts

```diff
@@ -306,13 +311,21 @@ export function finalizeChannel(channelId, state) {
 export function watcherChannelFinalized(channelId) {
     return async (
         dispatch,
-        getState,
+        _getState,
         { contractFactory }: IThunkDependencies
     ) => {
         try {
             const contract = await contractFactory.slotsChannelManagerContract()
-            const id = await contract.logChannelFinalized(channelId)
-            return await dispatch(actions.setChannelFinalized(id))
+            await contract
```

Comment:
await eventemitter?

Reply:
yes, inside the method we call an async method `getPublicAddress` from the KeyHandler